josharagon/LoL-Champion-Guide
# Single champion view

## What does this PR do?

- Adds router to the app
- Link each card to a single champion view component
- Load details about each character onto each page
- Make Description and background image dynamic for each character

## How should it be tested in the UI?

- clone repo
- npm start
- Click on any character and view their page. Click different ability or skin to see dynamic change.
